nuxt.config.ts

```diff
@@ -34,4 +34,10 @@ export default defineNuxtConfig({
     apiHost: process.env.PLAUSIBLE_API_HOST ?? 'https://plausible.io',
     trackLocalhost: true,
   },
+
+  app: {
+    head: {
+      link: [{ rel: 'stylesheet', href: 'https://unpkg.com/grapesjs/dist/css/grapes.min.css' }],
```

Comment:
Don't add style as a style tag, instead import the css file in the AppBlock component.